src/rendervulkan.cpp

```diff
@@ -3438,7 +3461,7 @@ bool vulkan_composite( const struct FrameInfo_t *frameInfo, std::shared_ptr<CVul
 	}
 	else
 	{
-		cmdBuffer->bindPipeline( g_device.pipeline(SHADER_TYPE_BLIT, frameInfo->layerCount, frameInfo->ycbcrMask(), 0u, frameInfo->colorspaceMask(), g_bOutputHDREnabled, g_bHDRForceWideGammutForSDR));
+		cmdBuffer->bindPipeline( g_device.pipeline(SHADER_TYPE_BLIT, frameInfo->layerCount, frameInfo->ycbcrMask(), 0u, frameInfo->colorspaceMask(), g_bOutputHDREnabled, g_bHDRForceWideGammutForSDR, g_bOutputHDREnabled ? g_bHDRItmEnable : false));
```

Comment:
hope this is ok
wasn't sure where to put this check 
this seemed the most logical place to me
works this way